Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java

```diff
@@ -246,8 +253,11 @@ public void onAdStart(@NonNull String placementReferenceId) {
                 mHandler.post(new Runnable() {
                     @Override
                     public void run() {
-                        mCustomEventInterstitialListener.onInterstitialShown();
-                        mCustomEventInterstitialListener.onInterstitialImpression();
+                        if (mInteractionListener != null) {
```

Comment:
minor: old name will appear at https://github.com/mopub/mopub-android-mediation/compare/master...Vungle:propagate-67x-to-develop#diff-b6527c4b838e3c383be81c1b1e94ccaaR242